code/modules/food_and_drinks/drinks/drinks_base.dm

```diff
@@ -1,30 +1,23 @@
 ////////////////////////////////////////////////////////////////////////////////
 /// Drinks.
 ////////////////////////////////////////////////////////////////////////////////
-/obj/item/reagent_containers/food/drinks
+/obj/item/reagent_containers/drinks
 	name = "drink"
 	desc = "yummy"
 	icon = 'icons/obj/drinks.dmi'
 	icon_state = null
 	container_type = OPENCONTAINER
-	consume_sound = 'sound/items/drink.ogg'
+	var/consume_sound = 'sound/items/drink.ogg'
```

Comment:
this needs to be moved below inherited vars to where `chugging` is

Reply:
Done!